src/vectorization/Vectorization.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright (C) 2019
+#      Max Planck Institute for Polymer Research & JGU Mainz
+#
+#  This file is part of ESPResSo++.
+#
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import espressopp
+from espressopp import pmi
+from espressopp.esutil import cxxinit
+import _espressopp
+
+r"""
+**************************************
+espressopp.vectorization.Vectorization
+**************************************
+
+.. function:: espressopp.vectorization.Vectorization(system, integrator, mode)
+
+    :param system: system object
+    :param integrator: integrator object
+    :param mode: (default='' equiv to 'SOA') 'SOA' for structure of arrays and 'AOS' for array of structures
+
+"""
+
+class VectorizationLocal(_espressopp.Vectorization):
+
+    def __init__(self, system, integrator, mode=""):
+        if pmi.workerIsActive():
+            if mode=="":
```

Comment:
This whole `if .... else ... if ... ` could be simplified. 
```python
if `mode == "" or mode == AOS:
    mode_int = _espressopp.VectorizationMode.AOS
elif mode == SOA:
    mode_int = _espressopp.VectorazationMode.SOA
else:
    raise ValueError('Incorrect mode [{}]'.format(mode))
```

Reply:
Simplified block.